src/tools/Makefile.am

```diff
@@ -126,6 +124,15 @@ pbs_python_LDADD = \
 	@KRB5_LIBS@
 
 pbs_python_SOURCES = \
+	$(top_builddir)/src/lib/Libpython/shared_python_utils.c \
+	$(top_builddir)/src/lib/Libpython/common_python_utils.c \
+	$(top_builddir)/src/lib/Libpython/pbs_ifl_wrap.c \
+	$(top_builddir)/src/lib/Libpython/pbs_python_external.c \
+	$(top_builddir)/src/lib/Libpython/pbs_python_svr_external.c \
+	$(top_builddir)/src/lib/Libpython/module_pbs_v1.c \
+	$(top_builddir)/src/lib/Libpython/pbs_python_svr_internal.c \
+	$(top_builddir)/src/lib/Libpython/pbs_python_svr_size_type.c \
+	$(top_builddir)/src/lib/Libpython/pbs_python_import_types.c \
```

Comment:
Why are you explicitly listing the source files rather than linking against the archive libraries? We use the same technique in src/server/Makefile.am

Reply:
I made `pbs_python` to compile with the libpython source files instead of linking to the archive library, so that the I can use the `PBS_PYTHON` flag to determine if my code in `pbs_python_external.c` is compiled with the server or with the `pbs_python` cmd. The code only need to execute when the interpreter is started by the server.

Reply:
Wasn't the point of having libpbspython.a and libpbspython_svr.a to compile them with different flags? Perhaps we shouldn't be referencing them both in pbs_python_LDADD.

Reply:
`libpbspython.a` and `libpbspython_svr.a` were compiled with different files. But what you pointed out is a good idea and I have modify the Makefiles so that `libpbspython_svr.a` is used exclusively for the server code.